gnocchi/rest/api.py

```diff
@@ -1651,6 +1651,61 @@ def groupper(r):
         return results
 
 
+# FIXME(sileht): should be in aggregates.api but we need to split all
+# controllers to do this
+def validate_qs(start, stop, granularity, needed_overlap, fill):
```

Comment:
i guess we could change this later but i imagine this is pretty restrictive to generically batch validate.